src/Cassandra.IntegrationTests/TestClusterManagement/TestClusterManager.cs

```diff
@@ -99,14 +105,50 @@ public static string DsePath
             get { return Environment.GetEnvironmentVariable("DSE_PATH"); }
         }
 
+        public enum BackendType
+        {
+            Hcd,
+            Dse,
+            Cassandra
+        }
+        
+        /// <summary>
+        /// "hcd", "dse", or "cassandra" (default)
+        /// </summary>
+        public static BackendType CurrentBackendType
+        {
+            get
+            {
+                string distribution = Environment.GetEnvironmentVariable("CCM_DISTRIBUTION") ?? "cassandra";
```

Comment:
Currently the test project is relying on checking for `DSE_VERSION` to see if the tests are running against DSE or not so we should also check if `DSE_VERSION` exists here before falling back to `cassandra`.

Reply:
Ok I see you also changed the implementation of `IsDse`. I still think we should check the `DSE_VERSION` env variable here before falling back to `cassandra`